NHLBI_BioData_Catalyst/python/ORCHID_COVID19_python.ipynb

```diff
@@ -98,14 +108,41 @@
   },
   {
    "cell_type": "markdown",
+   "id": "16dd2626",
    "metadata": {},
    "source": [
-    "# Packages Installation "
+    "# Packages Installation \n",
+    "**Note that if you are using the dedicated PIC-SURE environment within the BioData Catalyst Seven Bridges platform, the necessary packages have already been installed.**"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 1,
+   "id": "f8f425a7-68b7-4a56-9d15-d4917bf7b4e9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "# TERRA users uncomment the following line to specify package install location\n",
+    "sys.path.insert(0, r\"/home/jupyter/.local/lib/python3.7/site-packages\")"
```

Comment:
Should comment this line out to match the other notebooks

Reply:
Resolved in [acb3fe7a3d913a7140b9211676993e6ad22bc2d6](https://github.com/hms-dbmi/Access-to-Data-using-PIC-SURE-API/commit/acb3fe7a3d913a7140b9211676993e6ad22bc2d6)